pkg/cmd/dictionary/entries/import/import.go

```diff
@@ -0,0 +1,225 @@
+package importentries
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/spf13/cobra"
+
+	"github.com/algolia/cli/pkg/cmd/dictionary/shared"
+	"github.com/algolia/cli/pkg/cmdutil"
+	"github.com/algolia/cli/pkg/config"
+	"github.com/algolia/cli/pkg/iostreams"
+
+	"github.com/algolia/cli/pkg/prompt"
+	"github.com/algolia/cli/pkg/text"
+	"github.com/algolia/cli/pkg/utils"
+	"github.com/algolia/cli/pkg/validators"
+)
+
+type ImportOptions struct {
+	Config config.IConfig
+	IO     *iostreams.IOStreams
+
+	SearchClient func() (*search.Client, error)
+
+	DictionaryName    string
+	CreateIfNotExists bool
+	Wait              bool
+
+	File    string
+	Scanner *bufio.Scanner
+
+	ContinueOnError bool
+}
+
+// NewImportCmd creates and returns an import command for dictionary
+func NewImportCmd(f *cmdutil.Factory, runF func(*ImportOptions) error) *cobra.Command {
+	opts := &ImportOptions{
+		IO:           f.IOStreams,
+		Config:       f.Config,
+		SearchClient: f.SearchClient,
+	}
+
+	cmd := &cobra.Command{
+		Use:       "import <dictionary> -F <file> [--wait] [--continue-on-errors]",
+		Args:      validators.ExactArgs(1),
+		ValidArgs: []string{"stopwords", "plurals", "compounds"},
```

Comment:
Since this is already declared in the `shared` package, you could use `shared.DictionaryNames()` here for consistency.

Reply:
Good catch! thanks